src/app.js

```diff
@@ -4,17 +4,34 @@ const userDataRouter = require('./routes/userRoute');
 const config = require('./config/config');
 const errorHandler = require('./middlewares/errorHandler');
 const pageNotFoundHandler = require('./middlewares/pageNotFoundHandler');
+const Sentry = require('@sentry/node');
+const { nodeProfilingIntegration } = require('@sentry/profiling-node');
 
 const app = express();
 
+Sentry.init({
```

Comment:
I'm not sure, but probably you define the initialization in the config file and then import it
this will help you not to overload app.js